test/install-express-prompt.test.ts

```
@@ -7,151 +7,61 @@ import os from "node:os";
+      childPid = recordedPid;
+      expect(recordedPid).toBeGreaterThan(0);
+      expect(() => process.kill(recordedPid, 0)).toThrow(
+        expect.objectContaining({ code: "ESRCH" }),
+      );
+      childPid = undefined;
+    } finally {
+      if (childPid !== undefined) {
+        try {
+          process.kill(childPid, "SIGKILL");
+        } catch (error) {
+          if ((error as NodeJS.ErrnoException).code !== "ESRCH") throw error;
```

Biome (2.5.5)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)